File: C/Iniciante/1036/1036.c
#include <stdio.h>
#include <math.h>

int main() {
	double A, B, C;
	double delta, R1, R2;
	
	scanf("%lf %lf %lf", &A, &B, &C);
	
	if(A == 0)
		printf("Impossivel calcular\n");
		
	else{	
		delta = pow(B,2) - (4*A*C);
		
		if(delta>=0){
			R1 = (-B +sqrt(delta)) / (2*A);
			R2 = (-B -sqrt(delta)) / (2*A);
			
			printf("R1 = %.5lf\n", R1);
			printf("R2 = %.5lf\n", R2);
		}
		
		else
			printf("Impossivel calcular\n");
	}
	
	return 0;
}

File: Python/Iniciante/1036.py
# Problema 1036 - Beecrowd - Iniciante - Nível 3

# Importa o módulo com funções matemáticas básicas
import math

# Leitura dos valores
A, B, C = map(float, input().split())

# Verifica se existe raiz real para essa equação
if (pow(B, 2) - 4*A*C) > 0 and A != 0:
    # Cálculo das raízes
    R1 = (-B + math.sqrt(pow(B, 2) - 4*A*C))/(2*A)
    R2 = (-B - math.sqrt(pow(B, 2) - 4*A*C))/(2*A)

    # Imprime as raízes na saída padrão
    print(f"R1 = {R1:.5f}")
    print(f"R2 = {R2:.5f}")
else:
    # Caso não exista raíz real
    print("Impossivel calcular")
